content/en/security/application_security/threats/threat-overview.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Threat Overview
+kind: documentation
+aliases:  
+---
+
+{{< img src="security/application_security/threats/appsec-threat-overview-page-top.png" alt="Screenshot of the ASM Threat Overview page"  >}}
+
+## Introduction
+
+The ASM Threats Overview provides an at a glance view of your application and API posture, highlighting trends, service exposure, attack traffic, and the impact on business logic. Users can quickly pivot from widgets to related traces.
+
+The Threats Overview is divided into several sections, each focusing on a different aspect of security with supporting information.
+
+## Sections
+
+**Attack Surface Area**: This section provides insights into the exposed services, the tools attackers are using, and the commercial scanners that are identifying potential vulnerabilities.
+
+**Attack Traffic**: These graphs identify the classification of attacks, such as SSRF, LFI, SQL and command injection, allowing users to identify spikes in malicious traffic and patterns.
+
+**Business Logic**: This section focuses on the application-specific operations, such as account takeover attempts and custom business logic metrics, to help you understand trending attacks related to user accounts or fraud.
```

Comment:
Maybe we can cut it shorter like this?
**Business Logic**: This section focuses on fraud and business logic abuse such as account takeover attempts or any custom business logic events tracked by your application.

Reply:
+2